pydis_site/apps/api/models/bot/aoc_link.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+from pydis_site.apps.api.models.bot.user import User
+from pydis_site.apps.api.models.mixins import ModelReprMixin
+
+
+class AocAccountLink(ModelReprMixin, models.Model):
+    """An AoC account link for a Discord User."""
+
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        help_text="The user that is blocked from getting the AoC Completionist Role"
+    )
```

Comment:
This should be a `OneToOneField`. I realise this is using the Discord User model rather than the Django user model, but I believe this is still applicable https://docs.djangoproject.com/en/3.2/topics/auth/customizing/#extending-the-existing-user-model